prow/lib.sh

```diff
@@ -112,3 +112,33 @@ function clone_cni() {
       cd "${TMP_DIR}" || return
   fi
 }
+
+
+function check_and_install_kind() {
+  echo "Checking KinD is installed..."
+  if ! kind --help > /dev/null; then
+    if ! (go get sigs.k8s.io/kind); then
+      echo "Looks like KinD installation failed."
```

Comment:
echo real error why it failed to install

Reply:
i think the real error will be echoed because i didn't pipe the failure message to /dev/null